backend/config/database.js

```diff
@@ -5,27 +5,29 @@ const HOST = process.env.ABSTRACTION_DB_HOST
 
 const development = {
   dialect: 'mysql',
-  user: USER,
-  password: PASSWORD,
-  database: DB,
   host: HOST,
+  database: DB,
+  username: USER,
+  password: PASSWORD,
   port: 3306
 }
 
 const test = {
   dialect: 'mysql',
-  logging: false,
-  user: USER,
-  password: PASSWORD,
-  database: 'test',
   host: HOST,
-  port: 3306
+  database: 'test',
+  username: USER,
+  password: PASSWORD,
+  port: 3306,
+  logging: false
 }
 
-export default {
+// Additonal exports so that sequelize-cli can read this config properly
```

Comment:
`export default {}` is essentially syntactic sugar for `module.exports.default = {}`

You can still import this file like we do in others: `import dbConfig from 'config/database'`
That same statement in CommonJS format is `const dbConfig = require('config/database').default`

Because `sequelize-cli` reads in this file directly, it's expecting the modules to be exported using CommonJS format (`module.exports`). We might be able to hook Babel into the `sequelize-cli` process, but that seems unnecessary since this code already works.